test/utils/browser.js

```diff
@@ -30,32 +32,86 @@ exports.openMusic = function (options) {
   // Execute many async steps
   before(function startBrowser () {
     this.browser = wd.remote();
+    process.argv.forEach(function eachArgument (arg) {
+      // Uniquely identify a browserstack build based on timeout
+      if (arg === '40000') {
+        this.browser = wd.remote('hub.browserstack.com', 80);
+      }
+    }.bind(this));
```

Comment:
Instead of a magical argument being passed to our `argv`, how about we set another environment variable?

```js
// If we want to use BrowserStack, then point to it
if (process.env.USE_BROWSERSTACK) {
  this.browser = wd.remote('hub.browserstack.com', 80);
}
```